src/components/flat-table/flat-table.stories.mdx

```diff
@@ -198,6 +199,115 @@ Any cells preceding the `FlatTableRowHeader`s will automatically also be made st
   </Story>
 </Canvas>
 
+### Using row headers to create left and right hand side sticky columns
+
+In order to have right hand side sticky columns you should set the `FlatTableRowHeader` `stickyAlignment` prop to `right`.
+Any columns to following it will also be sticky.
+
+By default the left hand side columns have the prop set to `left`.
+It is important to make sure that the left hand side columns are not rendered before the right hand side ones.
+
+<Canvas>
+  <Story name="multiple sticky columns">
+    <FlatTable width="680px" overflowX="auto">
+      <FlatTableHead>
+        <FlatTableRow>
+          <FlatTableHeader>Sticky Column</FlatTableHeader>
+          <FlatTableRowHeader>Sticky Column</FlatTableRowHeader>
+          <FlatTableHeader>Scrollable Column</FlatTableHeader>
+          <FlatTableHeader>Scrollable Column</FlatTableHeader>
+          <FlatTableHeader>Scrollable Column</FlatTableHeader>
+          <FlatTableRowHeader stickyAlignment="right">Sticky Column</FlatTableRowHeader>
+          <FlatTableHeader>Sticky Column</FlatTableHeader>
+        </FlatTableRow>
+      </FlatTableHead>
+      <FlatTableBody>
+        <FlatTableRow>
+          <FlatTableCell>Sticky Content</FlatTableCell>
+          <FlatTableRowHeader>Sticky Content</FlatTableRowHeader>
+          <FlatTableCell>Scrollable Content</FlatTableCell>
+          <FlatTableCell>Scrollable Content</FlatTableCell>
+          <FlatTableCell>Scrollable Content</FlatTableCell>
+          <FlatTableRowHeader stickyAlignment="right">Sticky Content</FlatTableRowHeader>
```

Comment:
Do we want to allow consumers to make certain rows in a column sticky as opposed to the whole column? If not, what do you think about only passing `stickyAlignment="right"` in to `FlatTableRowHeader` in the head of the table so we don't have to repeat using the `stickyAlignment` prop for every row in the column?

Reply:
you would need to maintain that index and share it between the other rows, it becomes extremely messy to achieve. This component will be rewritten once the pending RFC has been accepted and this interface will change as a result to something like what you suggest where a single sticky index is set but that isn't easily achievable with current implementation